src/main/java/com/uber/cadence/worker/WorkerFactory.java

```diff
@@ -287,6 +289,16 @@ public synchronized void shutdownNow() {
     }
   }
 
```

Comment:
@AngerM-DD the comment I mentioned should also be put here  and the IWorkflowService for clarification. As these are the two places that users could look at.